src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -29,15 +29,13 @@ export type ClickableStyleProps<IComponent extends React.ElementType> = {
    */
   size?: 'sm' | 'md' | 'lg';
   /**
-   * Available _stylistic_ variations available for the ClickableStyle component
+   * Available _color_ variations available for the ClickableStyle component
    */
-  variant?:
-    | 'primary'
-    | 'secondary'
-    | 'tertiary'
-    | 'icon'
-    | 'link'
-    | 'destructive';
+  status?: 'brand' | 'neutral' | 'success' | 'warning' | 'alert';
```

Comment:
I'm not sure whether we should be using "alert" or "error". I know the utility tokens use "error" but some of the components are using "alert". I'd like for us to align on this and change the name everywhere, whichever direction is chosen. I'll open a ticket.